README.md

```diff
@@ -42,46 +42,176 @@ These elements are described in different folders. All the modelling happens wit
 
 The files that are outside from the `openfisca_country_template` folder are used to set up the development environment.
 
+## Packaging Your Country Package for Distribution
 
-## Installing
+If you wish to distribute your package via `pip install`, follow the steps given by the [Python Packaging Authority](https://python-packaging-user-guide.readthedocs.io/distributing/).
 
-> We recommend that you [use a virtualenv](https://doc.openfisca.fr/for_developers.html#create-a-virtualenv) to install OpenFisca. If you don't, you may need to add `--user` at the end of all commands starting by `pip`.
+If not, edit out the following sections:
+- the [Minimal Installation (pip install) section](#minimal-installation-pip-install) of the [Install instructions](#install-instructions);
+- the lines pertaining to the "minimal install" in the [section on serving the OpenFisca Web API](#serve-your-country-package-with-the-openFisca-web-api).
 
-To install your country package, run:
+## Install Instructions for Users and Contributors
 
+This package requires [Python 2.7](https://www.python.org/downloads/) and [pip](https://pip.pypa.io/en/stable/installing/) .
+
+Supported platforms:
+- GNU/Linux distributions (in particular Debian and Ubuntu);
+- Mac OS X;
+- Microsoft Windows (we recommend using [ConEmu](https://conemu.github.io/) instead of the default console).
+
+Other OS should work if they can execute Python and NumPy.
+
+### Setting-up a Virtual Environment with Pew
+
+We recommend using a [virtual environment](https://virtualenv.pypa.io/en/stable/) (abbreviated as "virtualenv") with a virtualenv manager such as [pew](https://github.com/berdario/pew).
+
+- A [virtualenv](https://virtualenv.pypa.io/en/stable/) is a project specific environment created to suit the needs of the project you are working on.
+- A virtualenv manager, such as [pew](https://github.com/berdario/pew), lets you easily create, remove and toggle between several virtualenvs.
+
+To install pew, launch a terminal on your computer and follow these instructions:
+
+```sh
+python -V # You should have python 2.7.9 or better installed on your computer.
+# If not, visit http://www.python.org to install it and install pip as well.
 ```
-pip install -e ".[test]"
+
+```sh
+pip install --upgrade pip
+pip install pew  # if asked, answer "Y" to the question about modifying your shell config file.
+```
+To set-up and create a new a virtualenv named **openfisca** running python2.7:
+
+```sh
+pew new openfisca --python=python2.7
+```
+
+The virtualenv you just created will be automatically activated. This means you will operate in the virtualenv immediately. You should see a prompt ressembling this (depending on the OS you are using) :
+```sh
+Installing setuptools, pip, wheel...done.
+Launching subshell in virtual environment. Type 'exit' or 'Ctrl+D' to return.
+openfiscabash-3.2$ 
```

Comment:
Same than `france`, this line depends on the shell you use. Users should not expect to find this line.